.github/workflows/deploy.yaml

```diff
@@ -2,8 +2,6 @@ name: Build and upload to PyPI
 
 on:
   push:
-    branches:
-      - master
```

Comment:
I removed lines 5&6 instead of 4–6 — I assume the `push:` still has to be present, @IAlibay

Reply:
Oops sorry yeah 5-6 (second time I mess up on this, accidentally told the same to @RMeli this morning 🤣 )